functions.php

```diff
@@ -71,11 +71,11 @@ function genesis_sample_enqueue_scripts_styles() {
 
 	$appearance = genesis_get_config( 'appearance' );
 
-	wp_enqueue_style(
+	wp_enqueue_style( // phpcs:ignore WordPress.WP.EnqueuedResourceParameters.MissingVersion -- see https://core.trac.wordpress.org/ticket/49742
```

Comment:
Thanks for linking to the Trac ticket. It [looks like](https://wordpress.org/support/topic/new-google-fonts-url-incompatible-with-wp_enqueue_style/#post-12761807) the `null` argument is a good approach.